graphql/schema/types/scraper.graphql

```diff
@@ -26,49 +26,7 @@ type Scraper {
     movie: ScraperSpec
 }
 
-type ScrapedScenePerformer {
-  """Set if performer matched"""
-  stored_id: ID
-  name: String!
-  gender: String
-  url: String
-  twitter: String
-  instagram: String
-  birthdate: String
-  ethnicity: String
-  country: String
-  eye_color: String
-  height: String
-  measurements: String
-  fake_tits: String
-  career_length: String
-  tattoos: String
-  piercings: String
-  aliases: String
-  tags: [ScrapedSceneTag!]
-
-  remote_site_id: String
-  images: [String!]
-  details: String
-  death_date: String
-  hair_color: String
-  weight: String
-}
-
-type ScrapedSceneMovie {
-  """Set if movie matched"""
-  stored_id: ID
-  name: String!
-  aliases: String
-  duration: String
-  date: String
-  rating: String
-  director: String
-  synopsis: String
-  url: String
-}
-
-type ScrapedSceneStudio {
+type ScrapedStudio {
```

Comment:
I'm wondering if all of these renames aren't an API breakage. Not from the "are the fields still available" perspective, but clients who use `__typename` won't work anymore I presume. But I can't think of any solution to this problem.

Reply:
Yeah, it's not ideal. I try to avoid breaking API in general, but I didn't really want these left hanging around. Clients will just have to deal with it I guess.